react/__mocks__/vtex.responsive-values/index.js

```diff
@@ -1 +1,2 @@
 export const useResponsiveValues = values => values
+export const useResponsiveValue = values => values
```

Comment:
Are these both necessary?

Reply:
Yep, they do slightly different things. 
https://github.com/vtex-apps/responsive-values
`useResponsiveValues` is used to process an array of values, while `useResponsiveValue` is for a single one